stage1/init/pod_test.go

```diff
@@ -50,8 +50,11 @@ func TestQuoteExec(t *testing.T) {
 			input:  []string{`path with "quo't'es" and \slashes`, `"arg"`, `\`},
 			output: `"path with \"quo\'t\'es\" and \\slashes" "\"arg\"" "\\"`,
 		}, {
-			input:  []string{`$path$`},
-			output: `"$path$"`,
+			input:  []string{`$path$`, `$argument`},
+			output: `"$path$" "$$argument"`,
+		}, {
+			input:  []string{`path`, `Args\nwith\nnewlines`},
```

Comment:
It could be done with
- `expect("hello\n.*world")` or
- `expect("hello(?s).*world")`.